worlds/subnautica/docs/setup_en.md

```diff
@@ -18,20 +18,26 @@
 
 4. Start Subnautica. You should see a Connect Menu in the topleft of your main Menu.
 
-## Troubleshooting
+## Connecting
 
-If you don't see the connect window check that you see a qmodmanager_log-Subnautica.txt in Subnautica, if not
-QModManager4 is not correctly installed, otherwise open it and look
-for `[Info   :   BepInEx] Loading [Archipelago 1.0.0.0]`, version number doesn't matter. If it doesn't show this, then
-QModManager4 didn't find the Archipelago mod, so check your paths.
+Using the Connect Menu in Subnautica's Main Menu you enter your connection info to connect to an Archipelago Multiworld.
+Menu points:
+ - Host: the full url that you're trying to connect to, such as `archipelago.gg:38281`.
+ - PlayerName: your name in the multiworld. Can also be called Slot Name and is the name you entered when creating your settings.
+ - Password: optional password, leave blank if no password was set.
 
-## Joining a MultiWorld Game
+After the connection is made, start a new game. You should start to see Archipelago chat messages to appear, such as a message announcing that you joined the multiworld.
 
-1. In Host, enter the address of the server, such as archipelago.gg:38281, your server host should be able to tell you
-   this.
+## Resuming
 
-2. In Password enter the server password if one exists, otherwise leave blank.
+When loading a savegame it will automatically attempt to resume the connection that was active when the savegame was made. 
+If that connection information is no longer valid, such as if the server's IP and/or port has changed, you can use the Connect Menu to connect to the new instance before loading the savegame.
```

Comment:
Does the game show you an error if the port changed, and an error if the port is valid but a different multiworld (hopefully invalid slot name or invalid game id for the slot name)? In a flowchart sort of way, it would be good to say how the user will know to back out and try the next steps.

Reply:
It will cause the menu to appear, allowing you to connect while the game is ongoing.